src/utils/Dimensions.ts

```diff
@@ -0,0 +1,33 @@
+module n3Charts.Utils {
+  'use strict';
+
+  export class Dimensions {
```

Comment:
yeah true, only accessing `Dimension.height` would make more sense for me than `Dimensions.height`